Lib/test/test_posix.py

```diff
@@ -1335,12 +1335,23 @@ def test_sched_getaffinity(self):
     def test_sched_setaffinity(self):
         mask = posix.sched_getaffinity(0)
         self.addCleanup(posix.sched_setaffinity, 0, list(mask))
+
         if len(mask) > 1:
             # Empty masks are forbidden
             mask.pop()
         posix.sched_setaffinity(0, mask)
         self.assertEqual(posix.sched_getaffinity(0), mask)
-        self.assertRaises(OSError, posix.sched_setaffinity, 0, [])
+
+        try:
+            posix.sched_setaffinity(0, [])
+            # gh-117061: On RHEL9, sched_setaffinity(0, []) does not fail
+        except OSError:
+            # sched_setaffinity() manual page documents EINVAL error
+            # when the mask is empty.
+            pass
+        except Exception as exc:
+            self.fail("unexpected exception: {exc!r}")
```

Comment:
It is not needed. Any exception is an error.

Reply:
Well, I wanted to raise an AssertionError explicitly. But ok, I will remove these 2 lines once the buildbot workers complete.

Two Centos9 workers are already green, whereas they are failing without this change.